.github/ISSUE_TEMPLATE/approved-reviewers-request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Fed. engineer PR approver checklist
+about: Checklist to ensure you are prepared to become a PR approver for your team's code
+title: ''
+labels: ['']
+assignees: ''
```

Comment:
How are we tracking if someone fills this out? Are we thinking a fed eng. comes through support and is handed this template to fill out? If that's the case, maybe this is fine and a label and assignee aren't needed.

Reply:
I ended up not changing anything here. We can make improvements if we need to later 🙂